workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/common/hooks/usePagination.tsx

```diff
@@ -5,9 +5,22 @@
 
 import {useState} from 'react';
 
-export default function usePagination() {
+export default function usePagination(urlParams?: URLSearchParams) {
+	if (urlParams && !urlParams.has('activepage')) {
+		urlParams.set('activepage', '1');
+	}
+
 	const [activeDelta, setActiveDelta] = useState<number>(20);
-	const [activePage, setActivePage] = useState<number>(1);
+	const [activePage, setActivePage] = useState<number>(
+		urlParams ? Number(urlParams.get('activepage')) : 1
+	);
+
+	const handlePageChange = (newPage: number) => {
+		if (urlParams) {
+			urlParams.set('activepage', `${newPage}`);
```

Comment:
```suggestion
			urlParams.set('activepage', newPage);
```

Reply:
It throws a type erro when i try to parse de number without casting to string.